tests/main/snap-user-service-start-on-install/task.yaml

```diff
@@ -1,5 +1,10 @@
 summary: Check that snap user services are started on install in existing sessions
 
+details: |
```

Comment:
Perhaps
    Check that sockets are created and activated correctly for snap user services when a
    service snap is installed and the experimental user-daemons flag is enabled